data/856/825/31/85682531.geojson

```diff
@@ -610,12 +610,8 @@
     ],
     "wof:breaches":[],
     "wof:concordances":{
-        "digitalenvoy:metro_code":[
-            276003
-        ],
-        "digitalenvoy:region_code":[
-            10478
-        ],
+        "digitalenvoy:metro_code":276003,
```

Comment:
If any one digitalenvoy concordance is a string, I think all these should be a string (which is generally our default recently.

Reply:
It looks like each code is a unique integer value.. though I can convert these to strings if there is a chance that these codes will become alphanumeric in the future.

Reply:
If they're all ints then ints here is fine.

Reply:
I'll add all commits in each repo's `stepps00/concordances` branch before merging so I can confirm that each value is actually an int.